### PR TITLE
colw/Add proposals loading indicator

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Add proposals loading indicator

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
changelog and unit tests are failing

File: src/components/governance/PageProposals.vue [hunk 1]
```diff
@@ -35,7 +35,9 @@
           @click.native="onPropose"
         />
       </div>
+      <TmDataLoading v-if="$apollo.loading" />
       <TableProposals
+        v-else
         :proposals="proposals"
         :loading="$apollo.queries.proposals.loading"
```
Comment (faboweb) [inline src/components/governance/PageProposals.vue]: is this used?

File: src/components/governance/PageProposals.vue [hunk 2]
```diff
@@ -35,7 +35,9 @@
           @click.native="onPropose"
         />
       </div>
+      <TmDataLoading v-if="$apollo.loading" />
```
Comment (faboweb) [inline src/components/governance/PageProposals.vue]: why not use the TmPage?

Comment (colw) [reply src/components/governance/PageProposals.vue]: I wanted the loading indicator underneath the 'Create Proposal' button. i.e. Allow the UI to be as functional as fast as possible.

Comment (faboweb) [reply src/components/governance/PageProposals.vue]: I get it! If this is the behavior we want, could be worth standardizing the page layout around it. putting the action buttons into one section and the data into another section.

**Comment by colw** [comment]
Resolutions are yarn only ☝️

File: package.json [hunk 3]
```diff
@@ -126,6 +126,7 @@
   },
   "resolutions": {
     "braces": ">=2.3.1",
-    "https-proxy-agent": ">=3.0.0"
+    "https-proxy-agent": ">=3.0.0",
+    "handlebars": ">=4.5.2"
```
Comment (jbibla) [inline package.json]: handlebars?

Comment (colw) [reply package.json]: https://handlebarsjs.com — `{{…}}` Templating syntax.

There was security vulnerability published 4 days ago: https://www.npmjs.com/advisories/1316

In any case whichever dependency (jest I think?) was using it has probably updated, so we don't need this resolution anymore.

File: src/components/governance/PageProposals.vue (hunk 1 above)
Comment (faboweb) [reply src/components/governance/PageProposals.vue]: @colw

Comment (colw) [reply src/components/governance/PageProposals.vue]: There is a `v-if` for loading above it?

Comment (colw) [reply src/components/governance/PageProposals.vue]: Not sure I understand.

Comment (faboweb) [reply src/components/governance/PageProposals.vue]: `:loading="$apollo.queries.proposals.loading"` is not used `TableProposals` doesn't have the property

Comment (colw) [reply src/components/governance/PageProposals.vue]: I see the loading on the TableProposals though is superfluous. I will remove that.

**Pull request merged**